source/usage-examples/deleteOne.txt

```diff
@@ -43,10 +43,8 @@ a Document Usage Example </usage-examples/findOne>`.
 Additional Information
```

Comment:
I can't comment, but on line 26, a period is missing at the end of the line. Also I don't think you need to have __ at the end of this type of link (but running a build should confirm this).